artemis-unit-test-support/src/main/java/org/apache/activemq/artemis/tests/extensions/PortCheckExtension.java

```diff
@@ -59,7 +59,7 @@ public static boolean checkAvailable(int port) {
       ServerSocket s = null;
       try {
          s = new ServerSocket();
-         s.bind(new InetSocketAddress("localhost", 61616));
+         s.bind(new InetSocketAddress("localhost", port));
```

Comment:
I brought this change alone into main... as this is not really related to your change.


if you could rebase it... it looks good to me.. merge it if tests are okay.